server/src/main/java/io/pravega/schemaregistry/service/SchemaRegistryService.java

```diff
@@ -843,8 +848,12 @@ private SchemaInfo normalizeSchemaBinary(SchemaInfo schemaInfo) {
                     break;
                 case Json:
                     schemaString = new String(schemaInfo.getSchemaData().array(), Charsets.UTF_8);
```

Comment:
So, you expect that the bytes are a result of UTF-8 encoding of the string, right? What'd happen if the schema was encoded in any other format?

Reply:
This was discussed in the main service side PR too. 
We make the assumption that json and avro schema strings are UTF8 encoded always.